tests/Tests/600__ai_apps/680__starburst/680__starburst.robot

```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Library         SeleniumLibrary
+Resource        ../../../Resources/ODS.robot
+Resource        ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Test Setup      Dashboard Test Setup
+Test Teardown   Dashboard Test Teardown
+
+*** Variables ***
+${SB_CARDS_XP}=     //article[@id="starburst"]
+${SB_BETA_DESC}=    //*[@class="pf-c-drawer__panel-main"]//div[@class="pf-c-alert pf-m-inline pf-m-info"]
+
+*** Test Cases ***
+Verify if the Starburst Beta text has been removed from Getting Started
+    [Documentation]
+
+    [Tags]  ODS-1158
+    Click Link  Explore
+    Wait Until Cards Are Loaded
+    Open Get Started Sidebar And Return Status    card_locator=${SB_CARDS_XP}
+    Page Should not Contain Element     xpath:${SB_BETA_DESC}
```

Comment:
hm to be consistent with the approach used in ODS-488 to do the check based on the RHODS version, I would do the same here. So, if RHODS version is greater than 1.8.0, then starburst must NOT have the beta desc. Else, it should.

This imply a change in the name of the test case

Reply:
Yes I have updated it. Can you review it again and tell me anything needs to be changed?

Reply:
the check on beta version should work in this way:
1. if RHODS >= v1.8.0 => starburst should not have Beta text (ODS-1158)
2. fi RHODS < v1.8.0 => starburst should have Beta text (ODS-605)

(pls avoid IF/ELSE in test cases. Have a look at the [style guide ](https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/)

Reply:
Actually I saw this kind of test cases that's why tried it. Anyways I have updated it.

Reply:
thanks!

there are still Test cases to be aligned with the new style guide, that's why